src/routes/index.ts

```diff
@@ -340,7 +341,9 @@ router
   });
 
 router.get(config.CHECK_YOUR_ANSWERS_URL, authentication, navigation.hasBOsOrMOs, checkYourAnswers.get);
+router.get(config.CHECK_YOUR_ANSWERS_WITH_PARAMS_URL, authentication, navigation.hasBOsOrMOs, checkYourAnswers.get);
```

Comment:
Changes to router for check your answers page will create conflict with Story for page changes ROE-2386

Reply:
Agreed. These changes should probably have been down as part of that story, but needed here to enable navigation to 'check your answers'. We can manage the conflict, whichever PR gets merged first.